pulp_rpm/tests/functional/constants.py

```diff
@@ -32,13 +32,22 @@
 RPM_UNSIGNED_FIXTURE_URL = urljoin(PULP_FIXTURES_BASE_URL, 'rpm-unsigned/')
 """The URL to a repository with unsigned RPM packages."""
 
-RPM_FIXTURE_COUNT = 39  # 35 Packages + 4 UpdateRecord units
+RPM_PACKAGES_FEED_COUNT = 35
```

Comment:
RPM_FIXTURE_PACKAGE_COUNT, just to keep things consistent.  Pulp 3 doesn't use "feed" anymore.